hazelcast/src/main/java/com/hazelcast/util/ResponseQueueFactory.java

```diff
@@ -173,5 +176,14 @@ public Object peek() {
                 lock.unlock();
             }
         }
+
+        /**
+         * Internal method, should be called under lock.
+         */
+        private Object getAndRemoveResponse() {
+            Object value = response;
+            response = null;
+            return (value == NULL) ? null : value;
```

Comment:
Because it's an expression and it's super helpful for my eyes to give a guidance, that it's not `return value`, but `return (expression which you have to read completely to understand what's going on)`. It's a pattern I always use, although it makes more sense if the first token is quite long.